src/commands/config/common/mod.rs

```diff
@@ -0,0 +1,135 @@
+use crate::{
+    commands::config::create,
+    config::{NetworkConfig, WalletConfig, ZKSyncConfig},
+};
+use dialoguer::{theme::ColorfulTheme, Confirm, Input, Select};
+use eyre::ContextCompat;
+use std::{path::PathBuf, str::FromStr};
+use zksync_ethers_rs::types::H160;
+
+pub const SELECTED_CONFIG_FILE_NAME: &str = ".selected";
+
+pub const DEFAULT_L1_RPC_URL: &str = "http://localhost:8545";
+pub const DEFAULT_L2_RPC_URL: &str = "http://localhost:3030";
+pub const DEFAULT_L2_EXPLORER_URL: &str = "http://localhost:3010";
+pub const DEFAULT_L1_EXPLORER_URL: &str = "";
+pub const DEFAULT_PRIVATE_KEY: &str =
+    "0x7726827caac94a7f9e1b160f7ea819f172f7b6f9d2a97f992c38edeab82d4110";
+// 0x36615Cf349d7F6344891B1e7CA7C72883F5dc049
+pub const DEFAULT_ADDRESS: H160 = H160([
+    0x36, 0x61, 0x5C, 0xf3, 0x49, 0xd7, 0xf6, 0x34, 0x48, 0x91, 0xb1, 0xe7, 0xca, 0x7c, 0x72, 0x88,
+    0x3f, 0x5d, 0xc0, 0x48,
+]);
+
+pub const CONFIG_OVERRIDE_PROMPT_MSG: &str = "Config already exists. Do you want to overwrite it?";
+pub const CONFIG_CREATE_PROMPT_MSG: &str = "This config does not exist. Do you want to create it?";
+pub const CONFIG_EDIT_PROMPT_MSG: &str = "What config do you want to edit?";
+pub const CONFIG_SET_PROMPT_MSG: &str = "What config do you want to set?";
+pub const CONFIG_DELETE_PROMPT_MSG: &str = "Are you sure you want to delete this config?";
+pub const CONFIG_SELECTION_TO_DELETE_PROMPT_MSG: &str = "What config do you want to delete?";
+pub const CONFIG_TO_DISPLAY_PROMPT_MSG: &str = "What config do you want to see?";
+pub const L1_RPC_URL_PROMPT_MSG: &str = "L1 RPC URL";
+pub const L2_RPC_URL_PROMPT_MSG: &str = "L2 RPC URL";
+pub const L2_EXPLORER_URL_PROMPT_MSG: &str = "L2 Explorer URL";
+pub const L1_EXPLORER_URL_PROMPT_MSG: &str = "L1 Explorer URL";
+pub const PRIVATE_KEY_PROMPT_MSG: &str = "Private key";
+pub const ADDRESS_PROMPT_MSG: &str = "Address";
+
+pub fn configs_dir_path() -> eyre::Result<std::path::PathBuf> {
+    let configs_dir_path = dirs::config_dir()
+        .ok_or_else(|| eyre::eyre!("Could not find user's config directory"))?
+        .join("zks-cli")
+        .join("configs");
+    if !configs_dir_path.exists() {
+        std::fs::create_dir_all(&configs_dir_path)?;
+    }
+    Ok(configs_dir_path)
+}
+
+pub fn config_path(config_name: &str) -> eyre::Result<std::path::PathBuf> {
+    Ok(configs_dir_path()?.join(&format!("{config_name}.toml")))
```

Comment:
Running clippy:
```warning: the borrowed expression implements the required traits
  --> src/commands/config/common/mod.rs:50:33
   |
50 | ...th()?.join(&format!("{config_name}.toml")))
   |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: change this to: `format!("{config_name}.toml")
```
```suggestion
    Ok(configs_dir_path()?.join(format!("{config_name}.toml")))
```

Reply:
It's weird, the CI passes and when doing `make cli` this doesn't appear. Nevertheless, it's a good catch.